File: docs/module-crates/python/my_module.py
from bots.hello_bot import BOT_NAME

def greet():
  print(f"Hi! I am hello bot")
  print(f"Hi I am {BOT_NAME}")

class Person:
  def __init__(self, name, age):
    self.name = name
    self.age = age

  def get_older(self, year):
    self.age += year

File: docs/module-crates/rust/src/my_module.rs
use crate::bots::hello_bot::BOT_NAME;

pub fn greet() {
    println!("Hi I am hello bot")
    println!("Hi! I am {}", BOT_NAME);
}

pub struct Person {
    pub name: String,
    age: i32,
}

impl Person {
    pub fn new(name: &str, age: i32) -> Self {
        Person {
            name: String::from(name),
            age: age,
        }
    }

    pub fn get_older(&mut self, year: i32) {
        self.age += year;
    }
}
